### ISSUE TITLE
Improve styling of the app

### ISSUE THREAD

**Issue by thtauhid** [thtauhid/terminal-portfolio#10]
Though the app somewhat looks like the Ubuntu terminal, some design modifications can still be done. 
This is a somewhat open-ended issue, you are not bound by my thoughts, feel free to implement your thoughts to align the design of the app with Ubuntu terminal.

**Comment by Ayansaxena24** [comment]
Pull Request Link --> https://github.com/thtauhid/terminal-portfolio/pull/32

Portfolio after updating :
![port](https://github.com/thtauhid/terminal-portfolio/assets/84512148/b09659d5-77b7-40ab-9086-fe1679c3d1fd)


Portfolio before updating :
![port 0](https://github.com/thtauhid/terminal-portfolio/assets/84512148/61354bf0-250d-4981-8e90-e01128973e07)


Since the portfolio was looking quite unattractive from UI point of view, I have updated the UI of the portfolio to some extend.
My changes :
1. Changed the Bg color (Added a gradient).
2. Added a Heading.
3. Added a Subheading.
4. Added a Hint.
5. Styled using Tailwind CSS.

I hope this will get approved for Hacktoberfest 2023!
